### PR TITLE
Docker dev

### PR DESCRIPTION
**Pull request by PaulGilmartin**

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -10,7 +10,16 @@
     'pgtrigger',
 ]
 # Database url comes from the DATABASE_URL env var
-DATABASES = {'default': dj_database_url.config()}
+DATABASES = {
```
Comment (romank0) [inline settings.py:13]: `dj_database_url` is convenient. If you want to override the URL you can specify the `default=` argument to `config` call like this:

```
dj_database_url.config(default="postgres://postgres:postgres@localhost:5432/postgres")
```

Comment (PaulGilmartin) [reply settings.py:13]: I did try this
```
DATABASES = {'default': dj_database_url.config(default="postgres://postgres:postgres@localhost:5432/postgres")}
```
but it always resulted in the error
```
E       django.db.utils.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
E       	Is the server running on that host and accepting TCP/IP connections?
E       connection to server at "localhost" (::1), port 5432 failed: Address not available
E       	Is the server running on that host and accepting TCP/IP connections?
```

File: docker-compose.yml [hunk 2]
```diff
@@ -10,8 +10,16 @@ services:
...
+      context: .
+      dockerfile: ./Dockerfile.dev
+    container_name: app
     depends_on:
       - db
+    stdin_open: true
+    tty: true
+    environment:
+      - DJANGO_SETTINGS_MODULE=settings
+      - ENVIRONMENT=development
```
Comment (romank0) [inline docker-compose.yml:23]: to fix the issue with pg connectivity and still use dj_database_url you need to keep the `DATABASE_URL=postgres://postgres:postgres@db:5432/postgres` here.

File: settings.py (hunk 1 above)
Comment (romank0) [reply settings.py:13]: The correct place to fix this is in the docker compose file as I've commented above

Comment (PaulGilmartin) [reply settings.py]: Nice! That worked :)

File: docker-compose.yml (hunk 2 above)
Comment (PaulGilmartin) [reply docker-compose.yml:23]: Thanks, that worked :)

**Comment by PaulGilmartin** [comment]
@romank0 How would you feel about the changes as they are now being committed?

**Comment by romank0** [comment]
> @romank0 How would you feel about the changes as they are now being committed?

I would put a line or two about how to use it into the docs otherwise this looks good.

**Pull request merged**